src/components/DateRangePicker.js

```diff
@@ -307,256 +414,359 @@ class DateRangePicker extends React.Component {
         </a>
         <div style={styles.container}>
           <div>
-            <div style={styles.optionsWrapper}>
-              {!this.state.showCalendar && (
-                <div>
-                  {this.props.showDefaultRanges &&
-                    <SelectionPane
-                      defaultRanges={this.props.defaultRanges}
-                      handleDefaultRangeSelection={this._handleDefaultRangeSelection}
-                      onDateBoxClick={(date, selectedBox) => {
-                        this.setState({
-                          currentDate: date || moment().unix(),
-                          selectedBox,
-                          showCalendar: !isLargeOrMediumWindowSize && true
-                        });
-                      }}
-                      selectedBox={this.state.selectedBox}
-                      selectedEndDate={this.props.selectedEndDate}
-                      selectedStartDate={this.props.selectedStartDate}
-                      styles={styles}
-                      theme={theme}
-                    />
-                  }
-                </div>
-              )}
-
-              {(this.state.showCalendar || isLargeOrMediumWindowSize) && (
-                <div>
-                  <div style={styles.calendarWrapper}>
-                    <div style={styles.calendarHeader}>
-                      <MonthSelector currentDate={this.state.currentDate} setCurrentDate={(currentDate) => this.setState({ currentDate, focusedDay: currentDate })} />
-                      <YearSelector currentDate={this.state.currentDate} setCurrentDate={(currentDate) => this.setState({ currentDate, focusedDay: currentDate })} />
-
-                    </div>
-                    <div style={styles.calendarWeek}>
-                      {[{ label: 'S', value: 'Sunday' },
-                        { label: 'M', value: 'Monday' },
-                        { label: 'T', value: 'Tuesday' },
-                        { label: 'W', value: 'Wednesday' },
-                        { label: 'T', value: 'Thursday' },
-                        { label: 'F', value: 'Friday' },
-                        { label: 'S', value: 'Saturday' }].map((day) => {
-                          return (
-                            <div key={day.value} style={styles.calendarWeekDay}>
-                              {day.label}
+            {this.state.showSelectionPane && (
+              <FocusTrap>
```

Comment:
If we're going to use `FocusTrap` here then we need to add a prop that can be passed to it to control its options like we do in the drawer and the modal

Reply:
This also needs to be the MXFocusTrap as well.